components/process.rst

```diff
@@ -410,6 +410,50 @@ instead::
     );
     $process->run();
 
+Executing a PHP child processes with the same configuration
+-----------------------------------------------------------
+
+.. versionadded:: 6.4
+
+    The `PhpSubprocess` helper was added in Symfony 6.4.
+
+When you start a PHP process, it's started using its default `ini` settings. Let's assume you have a configured
+`memory_limit` of `256M` in your `php.ini` and you want to disable it when running your `bin/console` script to access
```

Comment:
Minor comment: in the RST format that we use in docs, the code content is marked with two backticks instead of only one backtick as in Markdown:

```
Markdown: `memory_limit` of `256M` in your `php.ini`
     RST: ``memory_limit`` of ``256M`` in your ``php.ini``
```

Reply:
Done - I've also linked the class now.